ecs-init/gpu/nvidia_gpu_manager.go

```diff
@@ -169,14 +181,18 @@ func (n *NvidiaGPUManager) GetGPUDeviceIDs() ([]string, error) {
 		return nil, errors.Wrapf(err, "error getting GPU device count for UUID detection")
 	}
 	var gpuIDs []string
-	var i uint
-	for i = 0; i < count; i++ {
-		device, err := NvmlNewDeviceLite(i)
-		if err != nil {
-			seelog.Errorf("error initializing device of index %d: %v", i, err)
+	for i := 0; i < int(count); i++ {
+		device, ret := nvml.DeviceGetHandleByIndex(i)
+		if ret != nvml.SUCCESS {
+			seelog.Errorf("error initializing device of index %d: %v", i, nvml.ErrorString(ret))
```

Comment:
```suggestion
			seelog.Errorf("Error initializing device of index %d: %v", i, nvml.ErrorString(ret))
```

nit: log statements should start with a capital letter, applies to the log statement below as well.

Reply:
Fixed in [b036ab0](https://github.com/aws/amazon-ecs-agent/pull/4469/commits/b036ab03ae1e27d779f5f1558ef30a94338a57ac)